gpt_engineer/applications/cli/main.py

```diff
@@ -89,6 +89,20 @@ def load_env_if_needed():
     if os.getenv("ANTHROPIC_API_KEY") is None:
         load_dotenv(dotenv_path=os.path.join(os.getcwd(), ".env"))
 
+    if os.getenv("GOOGLE_API_KEY") is None:
+        load_dotenv()
+    if os.getenv("GOOGLE_API_KEY") is None:
+        load_dotenv(dotenv_path=os.path.join(os.getcwd(), ".env"))
+
+
+def model_env():
```

Comment:
Do as you consider it is efficient. I've made a separate function so that it's not called twice.

`load_env_if_needed()` is called on line 460 and `model_env()` on 298